docs/api/browser-window.md

```diff
@@ -662,6 +662,13 @@ Emitted when details regarding a requested resource is available.
 
 Emitted when a redirect was received while requesting a resource.
 
+### Event: 'page-favicon-set'
+
+* `event` Event
+* `favicons` [String]
+
+Emitted when page receives favicon urls.
```

Comment:
You can specify icons with various sizes, this method will fetch all favicon urls present for a site. Maybe i should provide `[size]` also as payload. I made the event name a bit confusing, will fix it and add api to get the current active [favicon](https://code.google.com/p/chromium/codesearch#chromium/src/content/public/browser/navigation_entry.h&sq=package:chromium&rcl=1429304361&l=154)